### ISSUE TITLE
Flux won´t clean up

### ISSUE THREAD

**Issue by SkaOArt** [Azanor/thaumcraft-beta#1583]
Hi i have a problem with the Flux Condenser and with the Lattice. I have a Clogged Lattice and right- clicking it with a essentia filter to have a normal lattice. Instantly the Lattice become a Clogged Lattice and the Flux doesn´t clean up. It doesn´t funktion it with a new Lattice, when i craft a new one an put it on the Condenser the same thing happend again. The Lattice become instandly to a clogged on and nothing happend in the aura. 
Has sombody the same Problem?

**Comment by androshalforc** [comment]
this has been an issue for some people it sounds like vitium is being sucked into your flux condonser for some reason. which seems to be a bug in some cases 

https://www.reddit.com/r/Thaumcraft/comments/9jav62/tc6_anyone_put_vitium_essentia_into_a_flux/

**Comment by SkaOArt** [comment]
Yeah that was the issue. I have no idea how Vitium get there but that was the problem. I am sure that i never put a vitium jar to the provide pipe. But who knows, maybe i forgotten. 
Thank you for you quick help.

**Comment by androshalforc** [comment]
some people have said that it seems to occasionally suck vitium from the output for some reason
